### PR TITLE
selenium fix

### PR DESCRIPTION
**Pull request by michalskrivanek**
- selenium: upgrade to 4.4.0
- selenium: export novnc port
- selenium: use Enter instead of clicking OK

### CODE REVIEW

File: ost_utils/pytest/fixtures/selenium.py [hunk 1]
```diff
@@ -116,12 +116,14 @@ def selenium_browser(
     container_id = ansible_storage.shell(
         "podman run -d"
         f" -p {selenium_port}:{selenium_port}"
+        "  -p 7900:7900"
```
Comment (tinez) [inline ost_utils/pytest/fixtures/selenium.py:119]: Why do we expose this port?

Comment (michalskrivanek) [reply ost_utils/pytest/fixtures/selenium.py:119]: from commit message:
for debugging we want to be able to get to port 7900 where selenium
starts a passwordless no-vnc instance

Comment (tinez) [reply ost_utils/pytest/fixtures/selenium.py:119]: ack

**Review by tinez — APPROVED** [review]

**Pull request merged**